### DISCUSSION TITLE
App compatibility list | App兼容列表

### DISCUSSION THREAD

**Discussion by CNflysky** [CNflysky/redroid-rk3588#8] [General]
Format:
- App name
- App package name
- App version
- App working status
- (Optional) Extra info
- (Optional) Screenshots

格式:
- App名称
- App包名
- App版本
- App运行情况
- （可选）额外信息
- （可选）截图

**Comment by CNflysky** [comment]
- App name: Minecraft
- App package name: com.mojang.minecraftpe
- App version: 1.21.1.03
- App working status: ✓
- Screenshot
![Screenshot_20240710-191619.png](https://github.com/CNflysky/redroid-rk3588/assets/48781081/37bc7182-aaef-4fe6-8544-0c6acc61802b)

**Comment by CNflysky** [comment]
- App name: Tiktok
- App package name: com.zhiliaoapp.musically
- App version: 35.5.4
- App working status: partial ⚠︎
- Extra info: I downloaded TikTok just to get the 15th anniversary cape for Minecraft. After registering an account, I didn't open TikTok for more than a month. Today I found out my account was banned. So use it on your own risk.
- Screenshot
![Screenshot_20240710-193153](https://github.com/CNflysky/redroid-rk3588/assets/48781081/2f0f19ff-c7ed-4aff-9d38-373e75d514f3)

**Comment by CNflysky** [comment]
- App name: X (Twitter)
- App package name: com.twitter.android
- App version: 10.48.0-release.0
- App working status: ✓
- Screenshot
![Screenshot_20240710-194238](https://github.com/CNflysky/redroid-rk3588/assets/48781081/b491292e-6167-451c-ae81-241dfe3dae46)

**Comment by CNflysky** [comment]
- App name: Discord
- App package name: com.discord
- App version: 236.20 - Stable
- App working status: ✓
- Screenshot
![Screenshot_20240710-194515](https://github.com/CNflysky/redroid-rk3588/assets/48781081/faf25dcd-cab3-4069-b5d4-c23381dd7488)

**Comment by CNflysky** [comment]
- App名称: 蔚蓝档案
- App包名:  com.RoamingStar.BlueArchive
- App版本: 1.8.2
- App运行情况: ✓
- 截图
![Screenshot_20240710-194949](https://github.com/CNflysky/redroid-rk3588/assets/48781081/a449fb34-47bb-44b0-8df2-dcac80ee8ee5)

**Comment by KylinDemons** [comment]
- App name: 碧蓝航线
- App package name: com.bilibili.azurlane
- App version: 8.1.1
- App working status: crashed(WIP)

**Reply by CNflysky** [reply]
> * App name: 碧蓝航线
> * App package name: com.bilibili.azurlane
> * App version: 8.1.1
> * App working status: crashed(WIP)

#3 soved.

**Comment by KylinDemons** [comment]
- App name: QQ
- App package name: com.tencent.mobileqq
- App version: 9.0.71
- App working status: ✓
- Screenshot
![image](https://github.com/CNflysky/redroid-rk3588/assets/41712927/aa97c63d-e681-46d8-8683-6209b02db755)

**Comment by KylinDemons** [comment]
- App name: Thanox
- App package name: github.tornaco.android.thanos
- App version: 5.0.8-prc
- App working status: ✓
- Screenshot
![image](https://github.com/CNflysky/redroid-rk3588/assets/41712927/f2628404-413c-4b4e-a12a-2bad4bc146b8)

**Comment by CNflysky** [comment]
- App name: 3dmark
- App package name: com.futuremark.dmandroid.application
- App version: 2.4.4934
- App running status: ✓
- Screenshot
![Screenshot_20240714-234723](https://github.com/user-attachments/assets/b7c2705b-04da-4ab4-9ac9-3824be6172f4)

**Comment by CNflysky** [comment]
- App name: Geekbench 6
- App package name: com.primatelabs.geekbench6
- App version: 6.3.0
- App working status: ✓
- Screenshots
![Screenshot_20240714-235119](https://github.com/user-attachments/assets/aea4f066-ef2f-4a34-b244-09a83b187f36)

**Comment by CNflysky** [comment]
- App name: Winlator
- App package name: com.winlator
- App version: 7.1
- App working status: ✓
- Screenshot
![Screenshot_20240714-235556](https://github.com/user-attachments/assets/08f153a4-4f45-46fa-930c-3c304adab661)

**Comment by li-chx** [comment]
App name: 明日方舟
App package name: com.hypergryph.arknights
App version: 2.3.81
App working status: ✓
Screenshot
![image](https://github.com/user-attachments/assets/6284f82a-0caf-4194-8e6c-0f6ded8ad64c)
